src/main/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/settings/ManagedIndexSettings.kt

```diff
@@ -109,5 +111,13 @@ class ManagedIndexSettings {
             Setting.Property.NodeScope,
             Setting.Property.Dynamic
         )
+
+        val ALLOW_LIST = Setting.listSetting(
+            "opendistro.index_state_management.allow_list",
+            ActionConfig.ActionType.values().toList().map { it.type },
+            Function.identity(),
```

Comment:
How is this ```Function.identity()``` used? Looking at the javadocs, it says that this argument is ```singleValueParser```

Reply:
Identity functions just return the value passed in. This is taken from examples in Elasticsearch using the Setting.listSetting. We aren't doing anything special with the strings in the list so just parsing and returning themselves should be fine.